nixos/modules/services/x11/desktop-managers/budgie.nix

```diff
@@ -39,6 +39,10 @@ let
     '';
     destination = "/share/gnome-background-properties/nixos.xml";
   };
+
+  budgie-control-center = pkgs.budgie.budgie-control-center.override {
+    enableSshSocket = config.services.openssh.startWhenNeeded;
+  };
```

Comment:
I doubt if hydra will build the `enableSshSocket = true;` one, probably fine considering the rebuild is small enough